app/views/content/is-teaching-right-for-me/all-subjects.md

```diff
@@ -0,0 +1,299 @@
+---
+title: Look at the subjects you could teach
+description: |-
+  Look at the subjects you could teach and choose what teacher training course you would like to do.
+
+keywords:
+  - teacher training
+  - teaching maths
+  - teaching history
+  - teaching science
+  - teaching art
+  - art and design
+
+
+---
+
+Explore the A-Z of all the subjects you could choose for your teacher training course.
+
+
+## Ancient Greek
+
+As an ancient Greek teacher you'll teach pupils about what life was like in ancient Greece, and the achievements of the people and their impact on the western world. 
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=A1&visa_status=false">Search for ancient Greek teacher training courses</a>.
+
+## Art and design 
+
+<a href="https://getintoteaching.education.gov.uk/funding-and-support/scholarships-and-bursaries?funding_widget%5Bsubject%5D=art_and_design&button=#funding-widget">Tax-free bursaries of £10,000 are available for eligible trainee art and design teachers</a>.
```

Comment:
@caitlin-reidy feels a bit strange to have a link about bursaries first? I think it would make more sense to have it after the line of intro about the subject so each one follows the same format. Unless we have some kind of box or tab that makes it look different? But as a link I'm not sure it works.

Also do we want the link to say Find out if you're eligible for a tax-free bursary of £10,000 to train to teach art and design?